config.yaml

```diff
@@ -228,6 +228,12 @@ postgresql_server:
   # THE PASSWORD WILL BE REMOVED FROM THE FILE AFTER THE INSTALLATION FINISHES
   postgres_password: ''
 
+  # postgresql.conf overrides
+  config:
+    shared_buffers: '1024MB'
+    effective_cache_size: '1024MB'
```

Comment:
on a 4GB server, it's not unlikely that 1GB will be left for the OS for fs buffers (this setting is mostly a best-effort guess, not a hard guarantee). "25% of RAM" is also the usual first estimate (eg. https://wiki.postgresql.org/wiki/Tuning_Your_PostgreSQL_Server )